src/shared/telemetry/telemetryUtils.ts

```diff
@@ -9,6 +9,19 @@ import * as vscode from 'vscode'
 import { ext } from '../extensionGlobals'
 import { Datum } from './telemetryEvent'
 
+export interface TelemetryName {
+    namespace: TelemetryNamespace
+    name: string
+}
+
+export enum TelemetryNamespace {
+    Cloudformation = 'cloudformation',
+    Credentials = 'aws_credentials',
```

Comment:
Similar to a comment I wrote elsewhere, if we use underscore as a namespace to name separator, should we consider not using the same separator within namespace values?

Reply:
✔️